src/s_user.c

```diff
@@ -1545,6 +1545,24 @@ send_msg_error(aClient *sptr, char *parv[], char *nick, int ret, aChannel *chptr
                    parv[0], nick);
 }
 
+/*
+ * Check if the recipient we are sending to is one of the services or stats addresses.
+ */
+static int is_aliastab_recipient(char *recipient)
+{
+    AliasInfo *ai;
+    char full_target[USERLEN + HOSTLEN + 2];
+
+    for(ai = aliastab; ai->nick; ai++)
+    {
+        ircsprintf(full_target, "%s@%s", ai->nick, ai->server);
+        if(!mycmp(recipient, full_target))
```

Comment:
Sorry, perhaps I'm missing something but are you thinking that only services PRIVMSG will contain the full_target format?

Reply:
It depends on SUPER_TARGETS_ONLY but I don't think it matters... he only wants to exempt services/stats from the  squelch block...